sdk/identity/identity/review/identity.api.md

```diff
@@ -104,6 +104,16 @@ export interface AzureDeveloperCliCredentialOptions extends MultiTenantTokenCred
     tenantId?: string;
 }
 
+// @public (undocumented)
+export class AzurePipelinesServiceConnectionCredential implements TokenCredential {
```

Comment:
I do think that `AzurePipelinesCredential` with a constructor that takes a required `serviceConnectionId` should be in consideration. 

I know we've gone back and forth on this design and will support this current design, but I think 

```ts
const myServiceConnectionId = "12345"
const credential = new AzurePipelinesCredential(clientId, tenantId, myServiceConnectionId)
```

is clear and offers a path to extending this via constructor overloads. Then, everything under the umbrella of "use this credential in Azure DevOps" can be bucketed and discovered under one name...

Reply:
To clarify, I am proposing renaming this credential to AzurePipelineCredential but otherwise the constructor LGTM

Reply:
plural :D